buildSrc/src/integTest/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleFuncTest.groovy

```diff
@@ -136,4 +142,47 @@ abstract class AbstractGradleFuncTest extends Specification {
             System.err.println("Syserr: " + proc.errorStream.text)
         }
     }
+
+    void setupRestResources(List<String> apis, List<String> tests = [], List<String> xpackApis = [], List<String> xpackTests = []) {
```

Comment:
I think this would better live in an AbstractRestResourcesFuncTest than in the general AbstractGradleFuncTest. `AbstractGradleFuncTest` should only contain more general logic applied to a most functional tests

Reply:
done: https://github.com/elastic/elasticsearch/pull/64186/commits/5021379eabe4a2362a763df91d342e8bbc3af495